platform-coop-toolkit/coop-map-directory-index
# feat: add a tool flow (resolves #37)

## Description

Adds flow for creating a new tool. Standardizes URL pattern for adding organizations, individuals, and tools to `/add/<item>/`.

## Steps to test

1. Check out branch.
2. Run migrations (no data alterations are made, just changes to the verbose names of some fields).
3. Navigate to `/add/tool.`

**Expected behavior:** Flow works as expected.

## Additional information

Not applicable.

## Related issues

Resolves #37.
